external/fv3fit/fv3fit/reservoir/domain.py

```diff
@@ -1,78 +1,10 @@
 import numpy as np
 
 
-def _slice(arr: np.ndarray, inds: slice, axis: int = 0):
+def slice_along_axis(arr: np.ndarray, inds: slice, axis: int = 0):
```

Comment:
I guess this is needed for future work?

Reply:
Yup, used in the cubedsphere subdomains.